frontend_tests/node_tests/stream_edit.js

```diff
@@ -32,6 +32,7 @@ mock_esm("../../static/js/components", {
     }),
 });
 
+const manage_subscribers = zrequire("manage_subscribers");
```

Comment:
What do you think about `stream_subscribers_ui.js` as a name for this module?  That feels more consistent, and I think it's helpful to have the `stream_` prefix to group it with all the other modules with that naming prefix.

Reply:
I'm kinda meh on it.  I get your logic, but `manage_subscribers` definitely fits my brain better.__

Reply:
I think I substantially prefer `stream_subscribers_ui.js` for consistency reasons; especially since `manage_subscribers` doesn't feel right for the case where we're creating a new stream -- I usually think of "Manage X" as an edit action.  Maybe start a #frontend discussion?  

I think it's worth trying to pick the name as we usually do over fixing forward because `git log` output is less nice with repeated renames to a set of code.

Reply:
I renamed it to `stream_subscribers_ui.js`.